database/factories/BookingFactory.php

```diff
@@ -32,9 +32,21 @@ public function definition()
             'amount'                => $this->faker->numberBetween(0, 10),
             'total_taxes'           => $this->faker->numberBetween(0, 10),
             'total_fees'            => $this->faker->numberBetween(0, 10),
-            'booking_category_id'   => BookingCategory::factory()->create()->id,
-            'rate_id'               => Rate::factory()->create()->id,
-            //Todo: Add morph factories
+            'booking_category_id'   => randomOrCreate(app('booking_category')),
+            'rate_id'               => randomOrCreate(app('rate')),
+            'experience_id'         => $this->faker->numberBetween(0, 10),
+            'experience_type'       => 'Game',
+            'order_id'              => $this->faker->numberBetween(0, 10),
+            'order_type'            => 'Order',
+            'booking_status_id'     => '1',
```

Comment:
@sl0wik I have a question about the `'booking_status_id'`

I saw the migration definition and saw this

![image](https://user-images.githubusercontent.com/13080491/110863986-7c2a1900-828f-11eb-9f41-47f9ad4b6bf6.png)

But I the whimsical there is this

![image](https://user-images.githubusercontent.com/13080491/110864171-bdbac400-828f-11eb-9095-6de295401f42.png)


I dont know very well how is the intention having the `Status::publishStatuses`

Reply:
@wolfrednicolas check out https://github.com/tipoff/bookings/pull/47, the decision was made to use Statuses package instead

Reply:
thanks @chx2